packages/mux-player/src/index.ts

```diff
@@ -81,14 +77,21 @@ class MuxPlayerInternal {
     this._deinitResizing();
   }
 
-  get _fragments() {
-    return this._template.fragments;
+  _setState(newState: Record<string, any>) {
+    Object.assign(this._state, newState);
+    this._render();
+  }
```

Comment:
the `setState` and `render` functions would be familiar to other UI lib devs.

also this `setState` could be a lot smarter with comparing state/props and queueing render calls but kept it dumb for now.